fews_3di/simulation.py

```diff
@@ -201,15 +213,19 @@ def _find_model(self) -> int:
 
     def _create_simulation(self, model_id: int) -> Tuple[int, str]:
         """Return id and url of created simulation."""
+        simulation_template = self.api.simulation_templates_list(
+            simulation__threedimodel__id=model_id
+        ).results[0]
```

Comment:
This raises an `IndexError` if there's no simulation template.
Is it handy to catch that exception and to provide a good error message? (It might not be needed).